owsi-core/owsi-core-components/owsi-core-component-jpa-more/src/main/java/fr/openwide/core/jpa/more/rendering/service/IRendererService.java

```diff
@@ -0,0 +1,24 @@
+package fr.openwide.core.jpa.more.rendering.service;
+
+import java.util.Locale;
+
+import fr.openwide.core.commons.util.rendering.IRenderer;
+import fr.openwide.core.context.IContextualService;
+import fr.openwide.core.jpa.more.util.fieldpath.model.FieldPath;
+
+
+public interface IRendererService extends IContextualService {
```

Comment:
You didn't split this into three interfaces, finally ? We spoke about splitting this into IRenderingService + IRendererRegistry + IConfigurableRenderingService (which inherits from the former two), same as ConfigurableConversionService or IConfigurablePropertyService. But maybe it's not relevant in this case, since the rendering service would return renderers anyway... Which would add a dependency to the IRenderer interface anyway.